tests/test.py

```diff
@@ -71,6 +71,22 @@ def test8_ods(self):
         for doc in documents:
             self.assertTrue(doc.startswith('http'))
 
+    def test9_kml(self):
+        maps = websearch.WebSearch('Madagascar').kml[:1]
+        #Verification de nombre de résultats
+        self.assertTrue(len(maps))
+        #verification lien
+        for map in maps:
+            self.assertTrue(map.startswith('http'))
+
+    def test10_custom(self):
+        documents = websearch.WebSearch('Biologie').custom_search('ps', 'application/postscript')[:1]
+        #Verification de nombre de résultats
+        self.assertTrue(len(documents))
+        #verification lien
+        for doc in documents:
+            self.assertTrue(map.startswith('http'))
```

Comment:
why you use map ? you mean doc, that's why test fail for custom_search

Reply:
sorry for the error